structure_test.go

```diff
@@ -115,6 +118,26 @@ func TestMain(m *testing.M) {
 
 	var err error
 
+	if pull {
```

Comment:
Should we error here if you pass pull and are not using the docker driver?

Reply:
yeah, I guess in my head I figured that would have been sort of implied but it's probably cleaner to just error explicitly here. updated